CMakeLists.txt

```diff
@@ -12,9 +12,11 @@ if(NOT MSVC)
 endif(NOT MSVC)
 
 if(APPLE)
-
+    set(ERLANG_INCLUDE_PATH "/usr/local/Cellar/erlang/23.0.2/lib/erlang/usr/include" CACHE PATH "Path to erlang includes")
```

Comment:
I think you can use `"/usr/local/lib/erlang/usr/include"` as the default - as @rbnpi pointed out in #5, this is actually linked to the specific installed version but should be stable between updates.

Reply:
@emlyn I had copied the erang files externally to this folder, thus enabling a fixed line in CMakeFiles.txt ie `"/usr/local/lib/erlang/usr/include"`
One possibility could be to use some code to search for the erlang headers on your system and then copy them to a specified location like this. I am looking at this. Otherwise some manual intervention is necessary dependent on your system. the brew location may be a good place to start. You can fix a brew install for a package NOT to update automatically when installing it and that could be helpful too.

Reply:
That folder also exists for me as a soft link into the specific version, and I haven't done anything manually to my install. I think Homebrew must have created it when I installed Erlang.